zerver/webhooks/beeminder/fixtures/derail.json

```diff
@@ -11,7 +11,7 @@
         "goal_type": "gainer",
         "autodata": null,
         "healthkitmetric": "",
-        "losedate": 1515759100,
+        "losedate": 1517759100,
```

Comment:
Could you please explain why you are changing this?

Reply:
In my view file, `time_remain` is ( losedate - current_time ) and I'm changing losedate only to make `time_remain` variable positive so that the screenshot doesn't show a negative time deadline :)
And so have to change that in `tests.py` (in line 6) also to match `time_remain` in view.py

Reply:
However it's not important, I thought to make screenshot more realistic ( by showing a positive time deadline)...😁